bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -407,6 +412,22 @@ void ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
   }
 }
 
+void ReplicaImp::sendInternalNoopsPrePrepareMsg(CommitPath firstPath) {
```

Comment:
This is one noop command, correct? Then we should get rid of the plural.

Reply:
fixed